Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs

```diff
@@ -1873,6 +1873,221 @@ public enum MessageBoxContentAreaType
 
     #endregion
 
+    #region Enum KryptonMessageBoxIcon
+
+    /// <summary>Specifies the icon type for <see cref="T:KryptonMessageBox"/>.</summary>
+    public enum KryptonMessageBoxIcon
```

Comment:
This needs a converter class as well,
So that the designer can display the text for the enum correctly

Reply:
Implemented `KryptonToastNotificationIconConverter` to handle this